src/isIsogram.test.js

```diff
@@ -6,4 +6,20 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`should return 'true' for correct isogram`, () => {
+    expect(isIsogram('playgrounds')).toBe(true);
```

Comment:
try to use .toBeFalsy() or  .toBeTruthy();

Reply:
> try to use .toBeFalsy() or .toBeTruthy();

Thanks, already fixed it